pkg/query-service/app/traces/v3/query_builder.go

```diff
@@ -302,7 +307,13 @@ func buildTracesQuery(start, end, step int64, mq *v3.BuilderQuery, tableName str
 		v3.AggregateOperatorRateAvg,
 		v3.AggregateOperatorRateMin,
 		v3.AggregateOperatorRate:
-		op := fmt.Sprintf("%s(%s)/%d", aggregateOperatorToSQLFunc[mq.AggregateOperator], aggregationKey, step)
+
+		rate := float64(step)
+		if options.PreferRPM {
```

Comment:
missing in the count operator?

Reply:
Count is different from rate. It is expected that count is commutatively calculated over a time interval.
Like count of time period 11:00-11:20 will always be less than or equal to 10:00-12:00.

Reply:
I didn't mean/say they are the same. Frontend has cps, cpm units https://github.com/SigNoz/signoz/blob/2bf534b56f6ca85c75ec6f85dd9e37a4210a89f0/frontend/src/container/NewWidget/RightContainer/alertFomatCategories.ts#L91-L97 so I was asking if the count should bother about this flag.

Reply:
counts/sec, ops/sec, requests/sec are all rate. So they should be covered with rate changes.